collector/info_schema_rocksdb_perf_context.go

```diff
@@ -0,0 +1,272 @@
+// Copyright 2018 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Scrape `information_schema.ROCKSDB_PERF_CONTEXT`.
+
+package collector
+
+import (
+	"context"
+	"log/slog"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const rocksdbPerfContextQuery = `
+                SELECT
+                  TABLE_SCHEMA, 
+                  TABLE_NAME, 
+                  ifnull(PARTITION_NAME, ''), 
+                  STAT_TYPE,
+                  VALUE
+                  FROM information_schema.ROCKSDB_PERF_CONTEXT
+                `
+
+// Metric descriptors.
+var informationSchemaRocksDBPerfContextMetrics = map[string]struct {
+	vtype prometheus.ValueType
+	desc  *prometheus.Desc
+}{
+	"USER_KEY_COMPARISON_COUNT": {
+		prometheus.CounterValue,
+		prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, informationSchema, "rocksdb_perf_context_user_key_comparison_count"),
+			"Total number of user key comparisons performed in binary search.",
+			[]string{"schema", "table", "part"}, nil,
```

Comment:
Can we refactor this to make the labels a var?

```go
var informationSchemaRocksDBLabels = []string{"schema", "table", "part"}
```

Reply:
done